environments.json

```diff
@@ -3341,14 +3372,30 @@
     "equals": "chrome89",
     "short": "OP 75",
     "release": "2021-03-24",
-    "obsolete": false
+    "obsolete": true
```

Comment:
i'm confused, why are we marking all these things as obsolete?

Reply:
AD1. Im restored the obsolete field to its previous value in this PR.
AD2. I marked all stable versions older by 2 than the newest as obsolete for the readability of the table